python/tests/test_schema.py

```diff
@@ -55,270 +60,194 @@ def test_table_schema_pyarrow_020():
-        }
-    )
-    assert pyarrow_field.name == field_name
-    assert pyarrow_field.type == pyarrow.list_(pyarrow.field("item", pyarrow.int32()))
-    assert pyarrow_field.metadata == metadata
-    assert pyarrow_field.nullable is False
-
-    field_name = "dictionary"
-    pyarrow_field = pyarrow_field_from_dict(
-        {
-            "name": field_name,
-            "nullable": False,
-            "metadata": metadata,
-            "type": {"name": "int", "bitWidth": 32, "isSigned": True},
-            "children": [],
-            "dictionary": {
-                "id": 0,
-                "indexType": {"name": "int", "bitWidth": 16, "isSigned": True},
-            },
-        }
-    )
-    assert pyarrow_field.name == field_name
-    assert pyarrow_field.type == pyarrow.map_(pyarrow.int16(), pyarrow.int32())
-    assert pyarrow_field.metadata == metadata
-    assert pyarrow_field.nullable is False
-
-    field_name = "struct_array"
-    pyarrow_field = pyarrow_field_from_dict(
-        {
-            "name": field_name,
-            "nullable": False,
-            "metadata": metadata,
-            "type": {"name": "list"},
-            "children": [],
-            "dictionary": {
-                "id": 0,
-                "indexType": {"name": "int", "bitWidth": 32, "isSigned": True},
-            },
-        }
-    )
-    assert pyarrow_field.name == field_name
-    assert pyarrow_field.type == pyarrow.map_(
-        pyarrow.int32(),
-        pyarrow.list_(
-            pyarrow.field(
-                "entries",
-                pyarrow.struct(
-                    [pyarrow.field("val", pyarrow.int32(), False, metadata)]
-                ),
-            )
+# def test_schema_pyarrow_from_decimal_and_floating_types():
+#     field_name = "decimal_test"
+#     metadata = {b"metadata_k": b"metadata_v"}
+#     precision = 20
+#     scale = 2
+#     pyarrow_field = pyarrow_field_from_dict(
+#         {
+#             "name": field_name,
+#             "nullable": False,
+#             "metadata": metadata,
+#             "type": {"name": "decimal", "precision": precision, "scale": scale},
+#         }
+#     )
+#     assert pyarrow_field.name == field_name
+#     assert pyarrow_field.type == pyarrow.decimal128(precision=precision, scale=scale)
+#     assert dict(pyarrow_field.metadata) == metadata
+#     assert pyarrow_field.nullable is False
+
+#     field_name = "floating_test"
+#     metadata = {b"metadata_k": b"metadata_v"}
+#     pyarrow_field = pyarrow_field_from_dict(
+#         {
+#             "name": field_name,
+#             "nullable": False,
+#             "metadata": metadata,
+#             "type": {"name": "floatingpoint", "precision": "HALF"},
+#         }
+#     )
+#     assert pyarrow_field.name == field_name
+#     assert pyarrow_field.type == pyarrow.float16()
+#     assert dict(pyarrow_field.metadata) == metadata
+#     assert pyarrow_field.nullable is False
```

Comment:
is this test still needed?

Reply:
Whoops! I had deleted but hadn't added those changes somehow. It's now deleted.